src/frontend/src/reducers/networks/utils/parsePathsData.ts

```diff
@@ -0,0 +1,101 @@
+// Refactored from latest Taui v3 parsePathsData to support new paths data format
+
```

Comment:
This updated code [ is from this comment](https://github.com/azavea/echo-locator/issues/628#issuecomment-3184615811), written when migrating to new analysis files in #628